portals/publisher/src/main/java/org/wso2/carbon/apimgt/ui/publisher/Util.java

```diff
@@ -149,6 +149,15 @@ public static String getServiceProviderTenantDomain(HttpServletRequest request)
         }
     }
 
+    public static boolean isEnableEmailUserName() {
+        boolean isEnableEmailUserName = Boolean.parseBoolean(CarbonUtils.getServerConfiguration().getFirstProperty("EnableEmailUserName"));
+        if (isEnableEmailUserName) {
+            return isEnableEmailUserName;
+        } else {
+            return false;
+        }
+    }
+
```

Comment:
Can't we use MultitenantUtils.isEmailUserName() instead of this particular util?

Reply:
If this is used this will be an external dependancy to the webapps. I implented the same approach we followed in the developer portal